test/unit/data/briefcase.js

```diff
@@ -58,7 +60,7 @@ describe('.csv.zip briefcase output @slow', () => {
     callAndParse(form, inStream, (result) => {
       result.filenames.should.eql([ 'mytestform.csv' ]);
       result['mytestform.csv'].should.equal(
-`SubmissionDate,name,age,hometown,KEY
+`SubmissionDate,name,age,hometown,KEY,SubmitterID,SubmitterName
```

Comment:
What do you think about including column headers for submitter ID and name only if that information was requested? Right now it seems like those headers are included whether or not extended metadata was requested.